app/script/entity/message/Message.js

```diff
@@ -350,4 +350,7 @@ z.entity.Message = class Message {
     }
     return false;
   }
-};
+}
+
+export default Message;
+z.entity.Message = Message;
```

Comment:
Is this still needed?

Reply:
Yes. All other message classes still extend from the globally scoped message class.